elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -60,7 +60,20 @@ done
 echo "List available plugins"
 ls -lrt $basedir
 
-# Download Knn lib 
+# Move RCA files
+## Move RCA folder
+cp -r $PACKAGE_NAME-$OD_VERSION/plugins/opendistro_performance_analyzer/performance-analyzer-rca $PACKAGE_NAME-$OD_VERSION
+chmod -R 755 elasticsearch ${PACKAGE_NAME}-${OD_VERSION}/performance-analyzer-rca
+## Move agent script directly into ES_HOME/bin
+mv $PACKAGE_NAME-$OD_VERSION/bin/opendistro_performance_analyzer/performance-analyzer-agent-cli $PACKAGE_NAME-$OD_VERSION/bin
+rm -rf $PACKAGE_NAME-$OD_VERSION/bin/opendistro_performance_analyzer
+## Create the rca_enabled.conf file
+mkdir -p ${PACKAGE_NAME}-${OD_VERSION}/data
+touch ${PACKAGE_NAME}-${OD_VERSION}/data/rca_enabled.conf
```

Comment:
could we use different folder for this, such as:/ES_HOME/\<rca\>/config/?

Reply:
btw, why do we need this file generated in the building process? could we have it in the artifact? what are the current files included in $PACKAGE_NAME-$OD_VERSION/plugins/opendistro_performance_analyzer/performance-analyzer-rca

Reply:
we can't use a different folder, the code currently expects them in these exact locations. we should be able to address this in the next release.

the file is generated in the build process and contains jars that we need to keep separate so that we don't run into classpath issues

Reply:
based on the comment from @ktkrg in the pr https://github.com/opendistro-for-elasticsearch/opendistro-build/pull/288, this rca_enabled.conf isn't required, could we remove it?

Reply:
I ran into issues with this when testing a standalone install.

I've removed the .conf file, but kept the logic to create the data folder. I'll test this with our artifacts and this branch.

Reply:
Thanks.